### PR TITLE
add use_distributed property

### PR DESCRIPTION
**Pull request by ZhiyuanChen**
https://github.com/huggingface/accelerate/issues/366

### CODE REVIEW

**Comment by ZhiyuanChen** [comment]
I noticed the rest modification proposed in the original issues have been resolved, so just add this last property

File: src/accelerate/accelerator.py [hunk 1]
```diff
@@ -273,6 +273,10 @@ def __init__(
         if self.rng_types is None:
             self.rng_types = ["torch"] if is_torch_version("<=", "1.5.1") else ["generator"]
 
+    @property
+    def use_distributed(self):
+        return self.distributed_type != DistributedType.NO
```
Comment (sgugger) [inline src/accelerate/accelerator.py:278]: Maybe also check for the number of processes to be `> 1` here, since you are replacing all those tests?

**Review by sgugger — APPROVED** [review]
Thanks for your PR! I left a small comment.

**Comment by ZhiyuanChen** [comment]
> Thanks for your PR! I left a small comment.

Thank you for the comments, I have updated the pr

**Pull request merged**